packages/block-library/src/navigation/block.json

```diff
@@ -54,6 +54,9 @@
 			"type": "string",
 			"default": "mobile"
 		},
+		"overlayTemplatePart": {
```

Comment:
I'd probably suggest just naming this attribute `overlay` or `overlayTemplate`, dropping the "Part" from the name